<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">Background-color: #000;

</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">Background-color: #000;

</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionIniciar_Captura"/>
   <addaction name="actionDetener_Captura"/>
   <addaction name="actionSalir"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/ic_exit_to_app_black_24dp_1x.png</normaloff>:/icons/resources/ic_exit_to_app_black_24dp_1x.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="toolTip">
    <string>Salir de la Aplicaci√≥n</string>
   </property>
  </action>
  <action name="actionIniciar_Captura">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/ic_photo_camera_black_24dp_1x (copia).png</normaloff>:/icons/resources/ic_photo_camera_black_24dp_1x (copia).png</iconset>
   </property>
   <property name="text">
    <string>Iniciar Captura</string>
   </property>
   <property name="toolTip">
    <string>Iniciar Captura</string>
   </property>
  </action>
  <action name="actionDetener_Captura">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/ic_photo_camera_black_24dp_1x.png</normaloff>:/icons/resources/ic_photo_camera_black_24dp_1x.png</iconset>
   </property>
   <property name="text">
    <string>Detener Captura</string>
   </property>
   <property name="toolTip">
    <string>Detener Captura</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/ic_people_black_24dp_1x.png</normaloff>:/icons/resources/ic_people_black_24dp_1x.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de...</string>
   </property>
   <property name="toolTip">
    <string>Acerca de...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
